spec/lib/reporting/proofing_rate_report_spec.rb

```diff
@@ -11,7 +11,7 @@
 
   describe '#as_csv' do
     before do
-      allow(report).to receive(:reports).and_return(
+      allow(report).to receive(:reduced_reports).and_return(
```

Comment:
I think we can undo these changes

Reply:
I should have included a note -- I only ran the MKMR spec before pushing this up. There's still a fair bit of WIP slop that needs fixing.